src/Controller/EnMarche/MembershipController.php

```diff
@@ -134,6 +137,7 @@ private function joinAdherent(Request $request): Response
         return $this->render('membership/join.html.twig', [
             'membership' => $membership,
             'form' => $form->createView(),
+            'formIsValid' => $isValid,
```

Comment:
As an alternative we can do `$isValid ?? null;` and then remove the line 122 which init the var.